clients/upscmd.c

```diff
@@ -28,8 +30,10 @@
 
 #include "upsclient.h"
 
-static char		*upsname = NULL, *hostname = NULL;
+static char			*upsname = NULL, *hostname = NULL;
 static UPSCONN_t	*ups = NULL;
+static int			status_info = 0;
+static unsigned int		timeout = 10;
```

Comment:
Similar to my comment about `DEFAULT_TIMEOUT`, I think this should be a `#define` that is used to generate the usage message.